### DISCUSSION TITLE
Centered on screen?

### DISCUSSION THREAD

**Discussion by UnclassedPenguin** [ddterm/gnome-shell-extension-ddterm#1130] [General]
I like to use Tilda, because you can make the terminal a percentage of height and width, and then center it in height and width as well so it shows up in the center of the screen. But tilda doesn't work with wayland. Is it  possible with yours? From what I could see in the settings it looked like you could only change the height.

**Comment by amezin** [comment]
Yes, only height (or width for left/right position)

**Comment by UnclassedPenguin** [comment]
Hey, thanks for the response. Is this something that might be a feature in the future?

**Reply by amezin** [reply]
https://github.com/ddterm/gnome-shell-extension-ddterm/issues/129
